src/alpine/3.17/helix/amd64/Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM amd64/alpine:3.17 as msquic
 
 # build MsQuic as we don't have packages
-RUN apk update && apk add --no-cache && \
-    apk add \
+RUN apk add --no-cache \
```

Comment:
Is there a reason we aren't using the `--upgrade` option like the dotnet-docker Dockerfiles?

https://github.com/dotnet/dotnet-docker/blob/c5d65d498e7ddacdd6a11410a7a2c5307a3e0bd1/src/runtime-deps/6.0/alpine3.19/amd64/Dockerfile#L11

Reply:
We haven't been using that pattern in this repo.  Is it necessary here?  I don't think the same reasons we do this in dotnet-docker applying here.  I see this change as being outside the scope of these changes.

Reply:
It would match the behavior of other package managers and potentially cut down on CVEs by installing the latest versions of packages if they're already installed from the base distro or another layer - https://github.com/dotnet/dotnet-docker/issues/4776.

If we want to address this in a separate issue or PR that's fine though.

Reply:
Alright you convinced me.  I included the option in this PR.